Sources/PrimerSDK/Classes/Data Models/OrderItem.swift

```diff
@@ -10,6 +10,7 @@
 import Foundation
 import PassKit
 
+@available(*, deprecated, message: "Set the order items in the client session")
```

Comment:
Not sure it's clear that the client session is something on the backend. Developers would first look at a SDK settings called "client session".

Maybe something like this?
`Set the order items in the client session with POST /client-session`

Reply:
We could also add a url link to the relevant part in the API documentation.

Reply:
Makes sense. I'll include the URL as well.